### ISSUE TITLE
[Accessibility]: Minor aria label cleanup

### ISSUE THREAD

**Issue by jasongornall** [sourcegraph/sourcegraph#36254]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34431

### Problem description

![image](https://user-images.githubusercontent.com/6912393/171051468-765c469c-fa05-408e-ae3a-e4e6d00bed1f.png)

Ark toolkit identified a couple missing aria elements. Should be able to lump these into 1 set of work. Contrast is a seperate ticket.

### Expected behavior

Errors should be zero from the Arc toolkit on the diff view
https://sourcegraph.com/github.com/sourcegraph/sourcegraph-code-discussions/-/commit/37873123e6fdf7706e6404f66d6a151e7d9a0cfa?visible=1#diff-b0678b51723ef8450026fa0f6967216fR22

 

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by gitstart-sourcegraph** [comment]
Hi @valerybugakov,
With reference to the attached page on issue description, running ARC toolkit generates a different set of results from the attached image. here is what it looks like on Prod and Local.

==> Prod
![Screenshot 2022-09-05 at 7 16 28 AM](https://user-images.githubusercontent.com/89894075/188372816-6fa6a369-2e23-443d-baf6-30452a8c638b.png)

i.e contains only contrast error and other alert/warnings. Perhaps these have been resolved in other issues.
Should we resolve the alerts as part of this PR? or mark this as resolved. Kindly confirm.

**Comment by valerybugakov** [comment]
Let's resolve this issue because the root causes have been fixed.
